patches/chrome-android-java-src-org-chromium-chrome-browser-toolbar-ToolbarManager.java.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+index 6c7a65a2dda292db027414d144b8565a7b0734cb..d7ff622e2b299a71d232241d0336c7b60dc19810 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+@@ -348,7 +348,7 @@ public class ToolbarManager
+     }
+ 
+     /** An {@link ObservableSupplier<Integer>} for the browser constraints of the current tab. */
+-    private static class ConstraintsProxy extends ObservableSupplierImpl<Integer>
```

Comment:
we can't do this with asm?

Reply:
@bridiver 
Indeed, my mistake. Reverted the way how it was before, through `((BraveTopToolbarCoordinator) mToolbar).getConstraintsProxy()`.

Fixed at https://github.com/brave/brave-core/pull/24227/commits/cdccb0920a98f030657edbd8973da67467b074d1 , thanks.